### PR TITLE
Support hmdb51 dataset preparation

### PR DESCRIPTION
**Pull request by dreamerlin**

### CODE REVIEW

File: tools/data/hmdb51/download_annotations.sh [hunk 1]
```diff
@@ -0,0 +1,20 @@
+#! /usr/bin/bash env
```
Comment (innerlee) [inline tools/data/hmdb51/download_annotations.sh:1]: this line looks suspicious. isn't it `#!/usr/bin/env bash`?

File: tools/data/hmdb51/download_annotations.sh [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+  echo "${DATA_DIR} does not exist. Creating";
+  mkdir -p ${DATA_DIR}
+fi
+
+cd ${DATA_DIR}
+wget http://serre-lab.clps.brown.edu/wp-content/uploads/2013/10/test_train_splits.rar --no-check-certificate
+
+# sudo apt-get install unrar
+unrar x test_train_splits.rar
+rm test_train_splits.rar
```
Comment (innerlee) [inline tools/data/hmdb51/download_annotations.sh:15]: it is very likely that user's pc does not have unrar installed. as a consequence, the newly downloaded data will be deleted without decompression. ref https://stackoverflow.com/questions/3474526/stop-on-first-error

File: tools/data/hmdb51/download_annotations.sh (hunk 1 above)
Comment (dreamerlin) [reply tools/data/hmdb51/download_annotations.sh]: Change it to `#!/usr/bin/env bash`. Same changes in other scripts are made in https://github.com/open-mmlab/mmaction2/pull/62

File: tools/data/hmdb51/download_annotations.sh (hunk 2 above)
Comment (dreamerlin) [reply tools/data/hmdb51/download_annotations.sh:17]: use `set -e` to make bash exit the script if any command fails

**Pull request merged**